src/components/filters/ConditionTypeFilterDropdown/index.tsx

```diff
@@ -4,36 +4,40 @@ import { ButtonSelectLight } from 'components/buttons/ButtonSelectLight'
 import { DropdownItem, DropdownPosition } from 'components/common/Dropdown'
 import { FilterDropdown } from 'components/pureStyledComponents/FilterDropdown'
 import { FilterTitle } from 'components/pureStyledComponents/FilterTitle'
+import { useWeb3ConnectedOrInfura } from 'contexts/Web3Context'
 import { ConditionType, ConditionTypeAll } from 'util/types'
 
 interface Props {
-  onClick: (value: ConditionType | ConditionTypeAll) => void
-  value: string
+  onClick: (value: ConditionType | ConditionTypeAll, filter: Maybe<string>) => void
+  value: Maybe<string>
```

Comment:
filter and value types could be more narrow?

Reply:
I think no